index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <!-- CSS Stylesheet-->
+        <link rel="stylesheet" type="text/css" href="src/css/style.css">
+        <!--Google fonts-->
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;600;700&display=swap" rel="stylesheet">
+        <!-- Font Awesome icons -->
+        <script src="https://kit.fontawesome.com/0e63fe95a0.js" crossorigin="anonymous"></script>
+        <title>Anghelo's Portfolio</title>
+    </head>
+    <body>
+        <nav class="nav-container">
+            <h4 class="nav-wordsColor">Luis Abarca</h4>
+            <div class="hidden-mobile">
+                <a class="nav-wordsColor" href="Portfolio">Portfolio</a>
+                <a class="nav-wordsColor" href="About">About</a>
+                <a class="nav-wordsColor" href="Contact">Contact</a>
+                <i class="far fa-envelope fa-inverse"></i>
+            </div>
+            <div class="shown-mobile">
+                <button class="ham-button" type="button">
+                    <i class="fas fa-bars justifyFlexcomponentsXY"></i>
+                </button>
+                <nav class="navbar">
+                    <a href="Portfolio"></a>
+                    <a href="About"></a>
+                    <a href="Contact"></a>
+                </nav>
+            </div>
+        </nav>
```

Comment:
- [ ] The navbar is a header of sorts so please wrap your navbar inside a `header` tag.